ca/config.go

```diff
@@ -93,14 +89,22 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte) error {
 // DefaultPolicy is the default policy used by the signers to ensure that the only fields
 // from the remote CSRs we trust are: PublicKey, PublicKeyAlgorithm and SignatureAlgorithm.
 var DefaultPolicy = func() *cfconfig.Signing {
+	return SigningPolicy(DefaultNodeCertExpiration)
+}
+
+// SigningPolicy creates a policy used by the signer to ensure that the only fields
+// from the remote CSRs we trust are: PublicKey, PublicKeyAlgorithm and SignatureAlgorithm.
+// It receives the duration a certificate will be valid for
+var SigningPolicy = func(certExpiry time.Duration) *cfconfig.Signing {
```

Comment:
cfssl uses this pattern to avoid concurrency errors when modifying default values.